packages/beacon-node/src/monitoring/service.ts

```diff
@@ -147,6 +148,10 @@ export class MonitoringService {
     const data: MonitoringData[] = [];
     const recordPromises = [];
 
+    if (this.options.collectSystemStats) {
```

Comment:
**Note:** it is intentional that `collectSystemStats` uses the term stats and `collectSystemData` the term data as the options value refers to not just collecting system data but also constructing and sending system stats as defined [here](https://github.com/ChainSafe/lodestar/blob/9ac65e1b94701c9f3856f85e7ab86bb61563951d/packages/beacon-node/src/monitoring/clientStats.ts#L197)